src/main/java/seedu/mark/model/reminder/Note.java

```diff
@@ -0,0 +1,70 @@
+package seedu.mark.model.reminder;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.mark.commons.util.AppUtil.checkArgument;
+
+/**
+ * Represents a note in reminder.
+ */
+public class Note {
+
+    public static final String INVALID_CHARACTER = "/";
+
+    public static final String MESSAGE_CONSTRAINTS = "Notes can contain any characters except " + INVALID_CHARACTER;
+
+    /*
+     * The first character of the note must not be a whitespace,
+     * otherwise " " (a blank string) becomes a valid input.
+     *
+     * The rest of the note can contain any character except the invalid character.
+     */
+    public static final String VALIDATION_REGEX = "[^\\s][^" + INVALID_CHARACTER + "]*";
+
+    public static final String DEFAULT_VALUE = "Open";
+    private String noteContent;
+
+    public Note(String noteContent) {
+        requireNonNull(noteContent);
+        checkArgument(isValidNote(noteContent), MESSAGE_CONSTRAINTS);
+        this.noteContent = noteContent;
+    }
+
+    /**
+     * Returns true if a given string is a valid note.
+     */
+    public static boolean isValidNote(String test) {
+        return test.matches(VALIDATION_REGEX);
+    }
+
+    /**
+     * Returns true if a given string is an empty note.
+     */
+    public static boolean isEmptyNote(String test) {
+        return test.trim().equals("");
+    }
+
+    @Override
+    public String toString() {
+        return noteContent;
+    }
+
+    /**
+     * Returns a {@code Note} with the default value.
+     */
+    public static Note getDefaultNote() {
```

Comment:
DefaultNote can just be stored as static variable so there is no need to insatiate a new object everytime

Reply:
But then there may have multiple reminder sharing the default note, if any of them edit the default reminder, the others will also changes. So I think it is still better to instantiate a default reminder every time.

Reply:
Yea you’re right. Sorry my bad